x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/config.tsx

```diff
@@ -0,0 +1,1385 @@
+    defaultMessage: 'hours',
+  });
+  const hour = i18n.translate('xpack.synthetics.monitorConfig.schedule.unit.hour', {
+    defaultMessage: 'hour',
+  });
+  switch (true) {
+    case value === 1:
+      unit = minute;
+      break;
+    case value / 60 === 1:
+      unit = hour;
+      break;
+    case value / 60 > 1:
+      unit = hours;
+      break;
+    default:
+      unit = minutes;
+  }
+
+  return i18n.translate('xpack.synthetics.monitorConfig.schedule.label', {
+    defaultMessage: 'Every {number} {unit}',
+    values: {
+      number: value >= 60 ? value / 60 : value,
+      unit,
+    },
+  });
+};
+
+const BROWSER_SCHEDULES = [
+  {
+    value: '3',
+    text: getScheduleContent(3),
+  },
+  {
+    value: '5',
+    text: getScheduleContent(5),
+  },
+  {
+    value: '10',
+    text: getScheduleContent(10),
+  },
+  {
+    value: '15',
+    text: getScheduleContent(15),
+  },
+  {
+    value: '30',
+    text: getScheduleContent(30),
+  },
+  {
+    value: '60',
+    text: getScheduleContent(60),
+  },
+  {
+    value: '120',
+    text: getScheduleContent(120),
+  },
+  {
+    value: '240',
+    text: getScheduleContent(240),
+  },
+];
+
+const LIGHTWEIGHT_SCHEDULES = [
+  {
+    value: '1',
+    text: getScheduleContent(1),
+  },
+  {
+    value: '3',
+    text: getScheduleContent(3),
+  },
+  {
+    value: '5',
+    text: getScheduleContent(5),
+  },
+  {
+    value: '10',
+    text: getScheduleContent(10),
+  },
+  {
+    value: '15',
+    text: getScheduleContent(15),
+  },
+  {
+    value: '30',
+    text: getScheduleContent(30),
+  },
+  {
+    value: '60',
+    text: getScheduleContent(60),
+  },
+];
+
+export const MONITOR_TYPE_CONFIG = {
+  [FormMonitorType.MULTISTEP]: {
+    id: 'syntheticsMonitorTypeMultistep',
+    'data-test-subj': 'syntheticsMonitorTypeMultistep',
+    label: 'Multistep',
+    value: FormMonitorType.MULTISTEP,
+    descriptionTitle: i18n.translate('xpack.synthetics.monitorConfig.monitorType.multiStep.title', {
+      defaultMessage: 'Multistep Browser Journey',
+    }),
+    description: i18n.translate(
+      'xpack.synthetics.monitorConfig.monitorType.multiStep.description',
+      {
+        defaultMessage:
+          'A lightweight API check to validate the availability of a web service or endpoint.',
+      }
+    ),
+    link: '#',
+    icon: 'videoPlayer',
+    beta: true,
+  },
+  [FormMonitorType.SINGLE]: {
+    id: 'syntheticsMonitorTypeSingle',
+    'data-test-subj': 'syntheticsMonitorTypeSingle',
+    label: 'Single Page',
+    value: FormMonitorType.SINGLE,
+    descriptionTitle: i18n.translate(
+      'xpack.synthetics.monitorConfig.monitorType.singlePage.title',
+      {
+        defaultMessage: 'Single Page Browser Test',
+      }
+    ),
+    description: i18n.translate(
+      'xpack.synthetics.monitorConfig.monitorType.singlePage.description',
+      {
+        defaultMessage:
+          'A lightweight API check to validate the availability of a web service or endpoint.',
```

Comment:
Change copy. Also see if copies for TCP and ICMP are correct.

Reply:
Adjusted. @colleenmcginnis Do you have specific copy for the descriptions for tcp and icmp monitors.